core/src/main/scala/kafka/cluster/Partition.scala

```diff
@@ -499,7 +500,15 @@ class Partition(val topicPartition: TopicPartition,
         addingReplicas = addingReplicas,
         removingReplicas = removingReplicas
       )
-      createLogIfNotExists(partitionState.isNew, isFutureReplica = false, highWatermarkCheckpoints)
+      try {
+        this.createLogIfNotExists(partitionState.isNew, isFutureReplica = false, highWatermarkCheckpoints)
```

Comment:
Do we need this? Ditto below.

Reply:
@junrao -- My understanding was we wanted to catch ZooKeeperClientException when performing createLogIfNotExists within makeLeader and makeFollower methods. Did we want to place this somewhere else?

Reply:
@andrewchoi5 : I was asking if we could change 

`this.createLogIfNotExists()`

to

`createLogIfNotExists()`

Reply:
@junrao  -- Got it. I misunderstood it for something else. Updated!